src/services/user/hooks/userService.js

```diff
@@ -88,6 +88,10 @@ const checkUniqueAccount = (hook) => {
 		});
 };
 
+const defineConsentStatus = (context) => {
+	const { data } = context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['data' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5276619)